plugins/package-managers/node/src/main/kotlin/pnpm/Pnpm.kt

```diff
@@ -129,7 +131,7 @@ class Pnpm(
         val json = PnpmCommand.run(workingDir, "list", "--json", "--recursive", "--depth", "Infinity", scopeOption)
             .requireSuccess().stdout
 
-        return parsePnpmList(json)
+        return parsePnpmList(json).flatten().toList()
```

Comment:
How about dropping `.toList()` here and instead change the return value to `Sequence<>`, just to avoid a conversion, even if we might not actually benefit from lazy evaluation of the sequence? I tried at it seems to build without further adjustments.